agent-lab/README.md

```diff
@@ -1,4 +1,4 @@
-# Agents Lab
+# Agent Lab
 
 This lab will provide you with a gentle introduction to AI Agents.
```

Comment:
it would be nice to have a link here:
```suggestion
This lab will provide you with a gentle introduction to [AI Agents](https://learn.microsoft.com/azure/cloud-adoption-framework/innovate/best-practices/conversational-ai).
```